manifests/agent/packages.pp

```diff
@@ -0,0 +1,9 @@
+class classroom::agent::packages {
+  $packages = [
+    'openssl',         # used for generating password hashes for user resources
```

Comment:
Can we always assume that the master will have the packages cached, for offline mode, though? Is there a better way to ensure they are, @samuelson @binford2k ?